test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl

```diff
@@ -218,6 +218,7 @@ inputs = {
   helm_exporter_github_owner        = "dfds"
   helm_exporter_repo_name           = "platform-manifests-qa"
   helm_exporter_repo_branch         = "main"
+  helm_exporter_target_namespaces   = "fluentd, flux-system, monitoring, traefik-blue-variant"
```

Comment:
Since the warnings (indicating it is detecting the configuration) showed up only after you removed the white spaces, should we not remove them here then?

Reply:
so I think whitespaces prevented helm exporter from working properly.. It also added the whitespaces into the log output  (whitespace next to each namespace name) like this:

```
time="2023-03-10T08:34:08Z" level=info msg="Watching namespace fluentd"
time="2023-03-10T08:34:08Z" level=info msg="Watching namespace  flux-system"
time="2023-03-10T08:34:08Z" level=info msg="Watching namespace  monitoring"
time="2023-03-10T08:34:08Z" level=info msg="Watching namespace  traefik-blue-variant"
```

Reply:
> so I think whitespaces prevented helm exporter from working properly..

Exactly, so shouldn't we remove them or am I missing something?